### DISCUSSION TITLE
Étude de cas : "Arts Plastiques"

### DISCUSSION THREAD

**Discussion by StachoB** [Hypertopic/HyperGlosae#136] [Show and tell]
https://github.com/Hypertopic/HyperGlosae/assets/74101149/e6e9063b-9f19-4a08-b67a-7f8313a5aedf

**Comment by benel** [comment]
Merci à toute l'équipe pour votre contribution à la collaboration avec l'@IMAJ-Troyes et au logiciel lui-même.

J'avais vu sur vos postes de travail, comme exemples d'images, les dessins des lauréats du concours sur plusieurs années. Il est dommage que vous ne les ayez pas utilisés pour la démonstration finale, ça aurait beaucoup aidé les parties prenantes à comprendre l'intérêt pour eux de ce que vous avez développé. Mais bon, on va tou•te•s faire un un effort d'imagination pour remplacer les vitraux et les diagrammes par des dessins d'enfants et de jeunes adultes...

**Comment by philacour** [comment]
Très intéressant.
Cela pemret de reconsidérer les images comme des documents et les collections comme des résultats de interprétatifs
Cela pourrait avoir des applications dans d'autres disciplines
PL
